### DISCUSSION TITLE
API to get active and execution history of workflow instance

### DISCUSSION THREAD

**Discussion by mhnvelu** [serverlessworkflow/synapse#5] [Ideas]
Hi,

I think it would be good to have below APIs from the Runtime:

1. Current status - can return currently active states
2. Get execution history of workflow instance - can return completed/failed state in workflow instance
3. Get workflow data input and output
4. Get input and output data of a state

Thanks

**Comment by cdavernas** [comment]
@mhnvelu not only will be shipped with the overhaul I'm currently working on, but we are also working on a Dashboard SPA
